### PR TITLE
[13.0][MIG] sale_fixed_discount

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @ForgeFlow

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM 👍

**Review by LoisRForgeFlow — APPROVED** [review]
code review :+1:

File: sale_fixed_discount/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Fixed Discount",
     "summary": "Allows to apply fixed amount discounts in sales orders.",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Sales",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (AdriaGForgeFlow) [inline sale_fixed_discount/__manifest__.py]: Could you take the chance to update the company name and any copyright in the module?

Comment (HviorForgeFlow) [reply sale_fixed_discount/__manifest__.py]: Sure!

**Comment by rousseldenis** [comment]
@HviorForgeFlow Do you plan to add a little test for the onchange ?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: functional test + code review

**Review by AdriaGForgeFlow — APPROVED** [review]

**Pull request merged**